libs/brackets-server/embedded-ext/project/main.js

```diff
@@ -32,6 +33,30 @@ define(function (require, exports, module) {
         .done(function(data) {
             _typeTable = data;
         });
+    // on opening of every project
+    ProjectManager.on("projectOpen", function () {
+        // if this is project with config.xml
+        var file = FileSystem.getFileForPath(ProjectManager.getProjectRoot().fullPath +"config.xml");
+        file.exists(function (err, exists) {
```

Comment:
Please do console log with error code and early return if err object is true.

Reply:
done